python/rascaline/rascaline/systems/__init__.py

```diff
@@ -32,6 +33,15 @@ def __init__(self):
     .. _chemfiles.Frame: http://chemfiles.org/chemfiles.py/latest/reference/frame.html
     """
 
+if HAVE_PYSCF:
```

Comment:
Could you make the PYSCF system into a separate PR?

Reply:
...bisecting only found the commit where equistore was renamed to metatensor, breaking the build on all older rascaline commits, due to a failed hash check

Reply:
> valgrind is annoyed at entries 5 and 9 of cargo test -p rascaline-c-api --test run-cxx-tests
(a problem that pre-dates my changes)

Hmm, can you open a separate issue for this? Or just share the setup with me on Slack? We run the valgrind tests on CI, so this is strange.

Reply:
well nevermind the error disappears for a fresh clone of master

Reply:
I'll need to look more into this, but when it does/doesn't happen seems to be nonsense for now.